program-runtime/src/prioritization_fee.rs

```diff
@@ -1,5 +1,5 @@
 /// There are 10^6 micro-lamports in one lamport
-const MICRO_LAMPORTS_PER_LAMPORT: u64 = 1_000_000;
+pub const MICRO_LAMPORTS_PER_LAMPORT: u64 = 1_000_000;
```

Comment:
minor: thoughts on putting the rounding function in here so we don't need to expose this unit.

My minor worry is making this publicly available someone will see and use it.

Reply:
Yea, that was my first choice too. Later I feel the round down logic is a business of compute_budget instruction, not prioritizatyion_fee's, in fact nothing about prioritization_fee is changed.

micro-lamport is defined in Terminologies, it is pretty much a public knowledge anyway.